tests/python/pytorch/sparse/test_matmul.py

```diff
@@ -30,6 +31,7 @@ def test_spmm(create_func, shape, nnz, out_dim):
     else:
         X = torch.randn(shape[1], requires_grad=True, device=dev)
 
+    X = rand_stride(X)
```

Comment:
This means we are only testing strided tensor while not testing normal tensors, though I feel it is a minor issue.

Reply:
Actually the stride is a random number in range [1, 4]. It is possible that the stride is one.

Reply:
Oh, we should avoid randomness in unit tests as much as possible. Otherwise, it is possible that some developers got lucky but later others fail the CI pipeline due to non-related errors. Please fix this.